### DISCUSSION TITLE
duplicate  the task manager set up for phone calls

### DISCUSSION THREAD

**Discussion by tmaleshafske** [TriliumNext/Trilium#1431] [Q&A] [answered]
Is there an easy way to duplicate the task manager to make a phone call manager?

I tried duplicating the structure last night and changing the needed labels to make it unique, but it would work.

What am I forgetting?

**Comment by zadam** [comment, marked as answer]
Easy way no, there is a "duplicate note sub-tree" task in a backlog, but for now you need to carefully duplicate it manually.

I assume it doesn't work because of some manual copying error ... :/

**Reply by tmaleshafske** [reply]
I have the majority of it duplicated, but it isn't creating notes in the other folders or in the Daily Journal
Could you look at my file and see where I screwed up.  Appreciate the help.
[database.zip](https://github.com/zadam/trilium/files/5563204/database.zip)

**Reply by tmaleshafske** [reply]
I got it!
5th or 6th time is a charm.  Here is what had to Duplicate Folder Structure, scripts, template. Next, I updated any references to #task to a new #foo, ensured the Root Folders were updated.  Lastly, I updated the scripts changing task to foo where needed.
